crates/solver/src/liquidity/order_converter.rs

```diff
@@ -55,8 +54,15 @@ impl OrderConverter {
             _ => (order.data.sell_amount, order.data.fee_amount),
         };
 
+        let id = match order.metadata.class {
+            OrderClass::Market => LimitOrderId::Market(order.metadata.uid),
+            OrderClass::Liquidity => {
+                LimitOrderId::Liquidity(LiquidityOrderId::ProtocolOrForeign(order.metadata.uid))
+            }
```

Comment:
Here I don't know how to make a difference between protocol and foreign liquidity order.
Either add this info to `Order` (which I don't want to do right now because `Order` is shared everywhere) or add a specialized implementation of function `normalize_limit_order` just for foreign liquidity orders which seems like an overkill right now.

Reply:
What needs to be handled differently across normal and foreign liquidity orders?

Reply:
In this function I just need to set the variant correctly.
Generally in the code there is no different handling anywhere (at least at the moment).
I just like having this classification as it helps with code readability (for example, before this PR I was not 100% what kind of liquidity orders are used in our app).

Reply:
Makes sense. 👍 
I see 2 more options:
1. pass another argument to the existing `normalize_limit_order()` function
2. manually update the `id` field of the normalized foreign liquidity order in `/http_solver/settlement.rs`

I don't really like any of those options TBH but I think I would go for `2`.

Reply:
Personally, I think (since the differentiation isn't important yet) that this can be left out for now until we need it.

When we do, I slightly prefer option 2 as well.

Reply:
Let's keep it as is (for now) since I don't find any of the options being a good improvement :/